man/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/conf.py.in ${CMAKE_CURRENT_BINARY_DIR}/conf.py @ONLY)
+
+add_custom_command(OUTPUT dnfdragora.1
```

Comment:
dnfdragora is a system administrative tool, so it should be in man page section 8.